AGENTS.md

```diff
@@ -8,22 +8,23 @@ Saleor Dashboard is a GraphQL-powered, single-page React application built with
 
 ### Basic Development
 
-- `npm run dev` - Start development server on port 9000 with host binding
-- `npm run build` - Build production bundle with 8GB memory allocation
+- `npm run dev` - Start development server on port 9000 with host binding, ALWAYS run this process in background, if you can't do that ask user
+- `npm run build` - Build production bundle
 - `npm run preview` - Preview production build locally
 
 ### Code Quality & Testing
 
+Before completing changes make sure you run these commands:
+
 - `npm run lint` - Run ESLint with auto-fix on src/ and playwright/ directories
-- `npm run test` - Run Jest tests for src/ directory
-- `npm run test:ci` - Run tests with coverage report
-- `npm run check-types` - Run TypeScript type checking for both src and playwright
-- `npm run check-types:src` - Type check src/ directory with strict plugin
-- `npm run check-types:playwright` - Type check playwright test files
+- `./node_modules/.bin/jest <file_path>` - Run specific test file (recommended for local development, running all tests is slow)
```

Comment:
Maybe we can have `jest: "jest"` script? it will accept arguments and alias path binary